build/deplicense.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
+#
+# Output the license info for the current directory's dependencies.
+#
+# The output is sorted by package name:
+#   <package-repo-root> <license info>
+
+dirs=()
+function visit() {
+    local dir="$1"
+    for dir in "${dirs[@]}"; do
+	if test "${dir}" = "${toplevel}"; then
+	    return 0
+	fi
+    done
+    dirs+=("${toplevel}")
+    return 1
+}
+
+function isApache2() {
+    grep -q '[[:space:]]*Apache License' "$1" && \
+	grep -q '[[:space:]]*Version 2\.0,' "$1"
+}
+
+function isBSD2Clause() {
```

Comment:
Nope. See http://opensource.org/licenses/BSD-3-Clause and http://opensource.org/licenses/BSD-2-Clause.